www/admin/inner.js

```diff
@@ -1696,17 +1715,23 @@ Example
         var _limit = APP.instanceStatus.defaultStorageLimit;
         var _limitMB = Util.bytesToMegabytes(_limit);
         var limit = getPrettySize(_limit);
-        var newLimit = h('input', {type: 'number', min: 0, value: _limitMB});
+        var newLimit = h('input', { type: 'number', min: 0, value: _limitMB, id: 'default-limit' });
         var set = h('button.btn.btn-primary', Messages.admin_setlimitButton);
+
         $div.append(h('div', [
             h('span.cp-admin-defaultlimit-value', Messages._getKey('admin_limit', [limit])),
             h('div.cp-admin-setlimit-form', [
-                h('label', Messages.admin_defaultLimitMB),
-                newLimit,
+                h('div', [
+                    h('label', { for: 'default-limit' }, 'default limit'), // XXX Messages.admin_defaultLimitMB is empty
```

Comment:
keep the translation key in place but add some text to it rather than introduce hard-coded text